newsroom/oauth_clients/clients_async.py

```diff
@@ -0,0 +1,53 @@
+import logging
+from pydantic import Field
+from datetime import datetime
+from typing_extensions import Optional, Annotated, List, Dict
+
+
+from superdesk.core.resources import ResourceModel, ResourceConfig, MongoResourceConfig, validators
+from superdesk.core.resources.service import AsyncResourceService
+from superdesk.core.web import EndpointGroup
+
+
+from content_api import MONGO_PREFIX
+
+
+class ClientResource(ResourceModel):
```

Comment:
Because we're using `ObjectId` for the `_id` of this model, we should probably extend from `ResourceModelWithObjectId` instead of `ResourceModel`

**ResourceModelWithObjectId** specifically states that we're using ObjectId

Reply:
Thanks for the update